composer.json

```diff
@@ -25,14 +25,15 @@
         "bref/bref": "^0.5.18 || ^1.0",
         "async-aws/sns": "^1.0",
         "async-aws/sqs": "^1.2",
-        "async-aws/event-bridge": "^1.0"
+        "async-aws/event-bridge": "^1.0",
+        "symfony/http-client": "<5.2"
```

Comment:
`async-aws` release a versin that fix it https://github.com/async-aws/core/releases/tag/1.7.0
Could you have a try?

Reply:
Thanks, that was quick! 😄 Done.